### PR TITLE
Updates for 501(c)(7) determination and some phrasing tweaks

### PR DESCRIPTION
**Pull request by kberzinch**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ seo:
 ---
 
 ## Purpose
-RoboJackets, Inc. is a Georgia non-profit corporation that is dedicated to the promotion, education, and advancement of the field of robotics. We support hands-on, super-curricular activities at the Georgia Institute of Technology. For more information about our programs, please visit our main website, [RoboJackets.org](https://robojackets.org).
+RoboJackets, Inc. is a Georgia non-profit 501(c)(7) corporation dedicated to the promotion, education, and advancement of the field of robotics. We support hands-on, super-curricular activities at the Georgia Institute of Technology. For more information about our programs, please visit our main website, [RoboJackets.org](https://robojackets.org).
```
Comment (kberzinch) [inline index.md:15]: It is a little awkward to work in tax-exempt status in this sentence. I was thinking about leaving it out; thoughts?

File: index.md [hunk 2]
```diff
@@ -37,13 +37,11 @@ The board of directors has regular meetings in the first month of each quarter.
 - [January 10th, 2021](https://drive.google.com/file/d/1Qn8lSbYW0-QlOJDbYjOnjvfZ577oi5n3/view)
 - [April 20th, 2021](https://drive.google.com/file/d/1Dzz-PpX8QZ9XXX7XP_Xx0P8qQtCpJMLX/view)
 
-## Internal Revenue Service Filings
-The corporation will be seeking tax exemption from the IRS under Section 501(c)(7). Public documents will be published here after each is approved by the IRS.
+## Internal Revenue Service Documents
+The IRS has determined we are exempt from federal income tax under IRC Section 501(c)(7). You may view our application and determination letter below.
```
Comment (kberzinch) [inline index.md:41]: IRC is intentional: it stands for Internal Revenue Code.

File: index.md (hunk 1 above)
Comment (JoshOldenburg) [reply index.md:15]: IMO it's fine as-is. Most people think non-profit and tax-exempt are the same and those who don't will probably recognize at least the prefix 501(c).

**Review by JoshOldenburg — APPROVED** [review]
Looks good to me

**Review by ryanstrat — APPROVED** [review]

**Pull request merged**